src/platform_impl/linux/common/xkb/state.rs

```diff
@@ -153,12 +153,12 @@ impl Drop for XkbState {
     }
 }
 
-/// Represents the current state of the keyboard modifiers
+/// Represents the current logical state of the keyboard modifiers
```

Comment:
Modifiers state is always `logical` since it's a logical concept.

Reply:
It's a logical concept with a 1-to-1 relationship to physical keys. They aren't all "just buttons" since buttons have labels and names, in this case directly matching the names of the logical state. So you need some mechanism to distinguish those two

Reply:
Sure, but we're not talking about all the buttons out there, just the ~8 very widespread modifier ones